tests/test_polydata.py

```diff
@@ -792,6 +792,20 @@ def test_lines():
     assert poly.n_cells == 1
 
 
+def test_strips():
+
+    # init with strips test
+    vertices = np.array([[0, 0, 0], [1, 0, 0], [1, 0.5, 0], [0, 0.5, 0]])
+    strips = np.hstack([[4, 0, 1, 3, 2]])
```

Comment:
No reason to have `hstack` here.
```suggestion
    strips = np.array([4, 0, 1, 3, 2])
```